base/applications/taskmgr/procpage.c

```diff
@@ -931,3 +936,300 @@ int CALLBACK ProcessPageCompareFunc(LPARAM lParam1, LPARAM lParam2, LPARAM lPara
     }
     return ret;
 }
+
+static BOOL DevicePathToDosPath(LPWSTR lpPath, DWORD dwLength)
+{
+    WCHAR *pszPath;
```

Comment:
`WCHAR *` --> `PWSTR` here and below (6 more changes).

Reply:
What's wrong with this?

Reply:
What's wrong with this?

Reply:
What's wrong with this?

Reply:
OK.

Reply:
What's wrong with this?

Reply:
OK.